Sources/CameraControlARView/RealityView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  RealityView.swift
+//  CompleteAnatomy
+//
+//  Created by Volodymyr Boichentsov on 11/10/2022.
+//  Copyright © 2022 3D4Medical, LLC. All rights reserved.
+//
+
+import SwiftUI
+import Combine
+import RealityKit
+
+fileprivate let arContainer = ARViewContainer.init(cameraARView: CameraControlARView.init(frame: .zero))
+var cancellables = [Cancellable]()
+
+public struct RealityKitView: View {
+    public typealias UpdateBlock = () -> Void
```

Comment:
Since this (`UpdateBlock`) is public, give me some detail of what it's being used for, and why a developer would include an updateBlock closure in their RealityKitView initializer. (Ideally, this would have documentation to explain it - but I'm happy to write it up if you can detail what the intent is for me)

Reply:
I'm going to remove alias :)